examples/api/lib/animation/curves/curve2_d.0.dart

```diff
@@ -2,27 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// Template: dev/snippets/config/templates/stateless_widget_material.tmpl
-//
-// Comment lines marked with "▼▼▼" and "▲▲▲" are used for authoring
-// of samples, and may be ignored if you are just exploring the sample.
-
 // Flutter code sample for Curve2D
-//
-//***************************************************************************
-//* ▼▼▼▼▼▼▼▼ description ▼▼▼▼▼▼▼▼ (do not modify or remove section marker)
-
-// This example shows how to use a [Curve2D] to modify the position of a widget
-// so that it can follow an arbitrary path.
```

Comment:
Why were descriptions deleted? They seemed like a useful brief summary of what the example shows

Reply:
That's a good question, there are two reasons: 

1) Because when these samples are presented in the API docs, they appear inside of an example box 
   which already displays the description, and a description at the top of the example code itself
   is ​redundant in that setting.

2) Because we want one source of truth for the description text, otherwise it will get stale. The
   ​description text already appears in the API documentation comments, which is a good place to
   have it for the code authors to see them in context.